src/SDKs/Dns/Dns.Tests/Dns.Tests.csproj

```diff
@@ -25,4 +25,7 @@
   <ItemGroup>
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Update="Microsoft.NET.Test.Sdk" Version="15.3.0" />
```

Comment:
@karravi-msft this has been long ignore for your RP. Please use latest version of ResourceManager nuget 1.6.0 (at least) and re-record your tests.
This has been long been ignored because there have been cases where your nuget has to meet some deadline.
Please use the latest version of Resource manager nuget for your testing your management client.

Reply:
resolved